intro-to-tests/intro-to-pytest.md

```diff
@@ -167,6 +167,24 @@ Read through these example `assert` statements as passing examples that prove we
 
 Tests should include at least one `assert` statement. There can be multiple assertions in a test.
 
+<!-- available callout types: info, success, warning, danger, secondary, star  -->
+### !callout-info
```

Comment:
Sorry if I read your question in the description wrong, but if you were considering if you wanted this new callout here or somewhere else in the lesson, I think here at the end of "Anatomy of a Unit Test" makes a lot of sense. 

The thought I had about it is whether the sentence sandwiched between this and the callout above would be easily overlooked. If we were concerned about that or having back to back callouts, we could consider placing the new callout just below the next section `## Arrange, Act, Assert is Useful Test Structure`, but I feel it fits best with the current section.

Reply:
I originally had it below the `## Arange, Act, Assert section`, but it didn't feel quite right, so I'll leave it here